### PR TITLE
Updated antisabotage to work with sniper rifles

### PR DESCRIPTION
**Pull request by realOneplustwo**
Antisabotage now applies to sniper rifles: if you shoot a node under your teammate, it does not break.

### CODE REVIEW

File: mods/pvp/sniper_rifles/init.lua [hunk 1]
```diff
@@ -157,6 +157,12 @@ local old_is_protected = minetest.is_protected
 local r = ctf.setting("flag.nobuild_radius") + 5
 local rs = r * r
 function minetest.is_protected(pos, name, info, ...)
+	if antisabotage.is_sabotage(pos, minetest.get_node(pos), minetest.get_player_by_name(name)) then
+		minetest.chat_send_player(name,
+			"You can't shoot blocks under your teammates!")
```
Comment (olliy1or) [inline mods/pvp/sniper_rifles/init.lua:162]: how about commenting this out and change the msg in https://github.com/MT-CTF/capturetheflag/blob/3e95d92359c7e93c8d43bcfe2db30ca529b651db/mods/pvp/antisabotage/init.lua#L21 to "You can't mine/shoot blocks under your teammates!"

**Review by Panquesito7 — COMMENTED** [review]
Receiving this error just right after trying to break a normal block with the sniper:

```
2021-02-12 14:00:43: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'default' in callback item_OnUse(): Runtime error from mod 'default' in callback node_on_dig(): ...n\..\games\capturetheflag\mods\pvp\antisabotage\init.lua:5: attempt to index local 'digg
2021-02-12 14:00:43: ERROR[Main]: er' (a nil value)
2021-02-12 14:00:43: ERROR[Main]: stack traceback:
2021-02-12 14:00:43: ERROR[Main]: 	...n\..\games\capturetheflag\mods\pvp\antisabotage\init.lua:5: in function 'is_sabotage'
2021-02-12 14:00:43: ERROR[Main]: 	...\..\games\capturetheflag\mods\pvp\sniper_rifles\init.lua:160: in function 'is_protected'
```

I likely have something messed up on my end.
Using Minetest Engine 5.4.0-dev.

**Comment by Panquesito7** [comment]
Still experiencing the same error, but this time `shooter_tweaks` is causing the problem; line 169.

**Comment by Panquesito7** [comment]
Well, I got it working now, but quite messy. I'll show you the changes once my version is working fine.

**Comment by realOneplustwo** [comment]
Since antisabotage has been removed, I think this PR should be closed.

**Pull request closed without merging**